lib/std/c/freebsd.zig

```diff
@@ -13,6 +13,8 @@ pub extern "c" fn getdents(fd: c_int, buf_ptr: [*]u8, nbytes: usize) usize;
 pub extern "c" fn sigaltstack(ss: ?*stack_t, old_ss: ?*stack_t) c_int;
 pub extern "c" fn getrandom(buf_ptr: [*]u8, buf_len: usize, flags: c_uint) isize;
 
+pub extern "pthread" fn pthread_getthreadid_np() c_int;
```

Comment:
```suggestion
pub extern "c" fn pthread_getthreadid_np() c_int;
```

we have "c" for other pthread stuff already. maybe it should be changed but how about that would be a separate change?